data/saas/dataset/firebase_auth_dataset.yml

```diff
@@ -0,0 +1,49 @@
+dataset:
```

Comment:
yeah, i'm with you on the `photo_url` - funny enough, i'd thought of including that in the dataset but then couldn't come up with a suitable fideslang data category to assign it, so then i just excluded it. but that's not a very good reason, so i think i'll include it again. any thoughts on what would be a good data category in the default taxonomy to assign it? (no worries if you don't have something off the top of your head)

the custom claims piece looks interesting - thanks for digging into that - but i'm just not sure how we'd handle that generically, since it's effectively schema-less. i _could_ see it containing relevant PII in certain use cases, so i don't feel great about just ignoring it

if we did decide to support this, what i'm thinking is we could allow users to configure their dataset according to what their `custom_claims` metadata looks like. then i could probably have something generic in the function to dynamically support whatever fields the dataset defines in the sub-object.

that's not a trivial amount of work, though, and we don't have a great paradigm in place for allowing users to overwrite the default datasets we're providing them, which would be necessary for this to work. so i'm a bit unsure here.

Reply:
I usually go with a more generic data category if a specific one isn't available, in this case I would lean towards just annotating `photo_url` as `user` data.

Reply:
@earmenda and i discussed offline that we won't support `custom_claims` in our first iteration, but if it turns out to be an important client need, we can look to implement it.

we'd probably look to support it by either writing some code into our function to dynamically traverse/enumerate the subfields within the `custom_claims` field as declared in the dataset. we could also likely just serialize and dump out the `custom_claims` subobject as a JSON blob in an access request. we can look to explore those if/when the need arises.